source/message.html

```diff
@@ -4,24 +4,66 @@
 <TITLE></TITLE>
 <SCRIPT LANGUAGE=javascript>
 <!--
-function escapeKeyPress(e) {
+var args = window.dialogArguments;
+
+function handleKeyPress(e) {
 	e = e || window.event;
 	if(e.keyCode == 27){ // code for escape
 		window.close();
+	} else if (e.altKey && e.keyCode === 67) {  // Code for Alt+c for copy button
+		onCopyButtonPress();
+	}
+}
+
+function onCopyButtonPress() {
+	// Copy code came from http://www.codestore.net/store.nsf/unid/DOMM-4QHQE8/
+	var rng = document.body.createTextRange();
+	rng.moveToElementText(messageID);
+	rng.scrollIntoView();
+	rng.select();
+	var success = rng.execCommand("Copy");
+	rng.collapse(false);
+	rng.select();
+	if (args) {  // Notify the user about the copy result
+		if (success) {
+			alert(args.item('copySuccessfulAlertText'));
+		} else {
+			alert(args.item('copyFailedAlertText'));
```

Comment:
where is this text coming from? I don't see any other references

Reply:
Fixing in suggestion https://github.com/nvaccess/nvda/pull/16369/files#r1654532903

Meant to add that over the weekend, but I got called away unexpectedly and am only now back.